guides/voice/gather-dtmf-tones-guide/gather-example-step-1/example.5.x.cs

```diff
@@ -0,0 +1,49 @@
+// In Package Manager, run:
+// Install-Package Twilio.Mvc -Pre
+
+using System.Web.Mvc;
+using Twilio.TwiML;
+
+public class VoiceController : Controller
+{
+	[HttpPost]
+	public ActionResult Index(string digits)
+	{
+		var response = new VoiceResponse();
+
+		if (!string.IsNullOrEmpty(digits))
+		{
+			switch (digits)
+			{
+				case "1":
+					response.Say("You selected sales. Good for you!");
+					break;
+				case "2":
+					response.Say("You need support. We will help!");
+					break;
+				default:
+					response.Say("Sorry, I don't understand that choice.").Pause();
+					RenderMainMenu(response);
+					break;
+			}
+		}
+		else
+		{
+			// If no input was sent, use the <Gather> verb to collect user input
+			RenderMainMenu(response);
+		}
+
+		return Content(response.ToString(), "text/xml");
+	}
+
+	// helper function to set up a <Gather>
+	private static void RenderMainMenu(VoiceResponse response)
```

Comment:
Thanks, usually I don't think is a good idea to include the data type in the method's name. Although I couldn't agree more on removing that comment. There is actually an issue in this snippet, `VoiceResponse` is not longer available in the Twilio helper library, so it has to be changed.